cvmfs/cloud_testing/run_mac.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -x
+# in this script we will sum up all the different steps to run the tests
+# on a mac VM from a mac host
+SCRIPT_LOCATION=$(cd "$(dirname "$0")"; pwd)
+cd "$SCRIPT_LOCATION"
+
+testee_url=$1
+package_name=$2
+source_name=$3
+osx_name=$4  # might be yosemite, el_capitan, etc.
+setup_script=$5
+run_script=$6
+client_package_url="$testee_url/$package_name"
+source_tarball_url="$testee_url/$source_name"
```

Comment:
I'd copy over the parameter handling along with the "usage"-string from `cvmfs/cloud_testing/run.sh` and adapt it. Just for consistency and self-documentation.